src/ManifestGenerator/manifestgen.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
```

Comment:
I didn't put the appropriate copyright header in the original--that should be required, right?

Reply:
i have modified your original README.md file and added # License which covers Copyright. so this should be covered.